### ISSUE TITLE
How to check whether a person pressed the Ok button or the cancel button in CTkInputDialog?

### ISSUE THREAD

**Issue by shasankp000** [TomSchimansky/CustomTkinter#836]
I tried this

``` 
self.confirm_dialog = ctk.CTkInputDialog(text="Please Enter your password. It won't be saved.", title="Confirm account.")

        if self.confirm_dialog._ok_event():
            password = self.confirm_dialog.get_input()

            # Do something else

        else:
            pass
```

But ended up with some errors.

**Comment by ArielMAJ** [comment]
I just came here to look this up too. I recently realized the code I'm working on isn't checking whether the user is pressing "ok" or "cancel" and I can't figure out how to do it. No matter what the user presses, I always am able to get the input (resulting in a bug).

**Comment by sternflyer** [comment]
There is a bug in the code. See #853
With that fix, get_input() will return None if Cancel was clicked.

**Comment by ArielMAJ** [comment]
That seems to be it @sternflyer. Hopefully #853 will be accepted soon.

**Comment by GameCX** [comment]
![image](https://user-images.githubusercontent.com/108629054/215290190-0964a395-96dd-4367-8968-95ca6b78d29c.png)

Please fix the bug

**Comment by ElectricCandlelight** [comment]
There is already a pull request for this issue

**Comment by TomSchimansky** [comment]
resolved with version 5.2
